graylog2-server/src/main/java/org/graylog2/indexer/datastream/policy/actions/RolloverActionFormatter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.indexer.datastream.policy.actions;
+
+import static org.graylog2.shared.utilities.StringUtils.f;
+
+public class RolloverActionFormatter {
```

Comment:
maybe we should rename this class to something more general as it can be used for all formats in the ism policy, something like `PolicyFormatUtils`?

Reply:
Thanks for the tip. We now have respectively named formatter enums in https://github.com/Graylog2/graylog2-server/pull/18473#discussion_r1515300637. Let me know if you see any possible improvements in the naming there.